### PR TITLE
Move marketing pages to Borrow

### PR DESCRIPTION
**Pull request by ferni**
# [Move marketing pages to Borrow](https://app.clubhouse.io/oazo-apps/story/2117/moving-marketing-pages-to-borrow)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- Imported pages: privacy, support, about, careers, careers/[slug] (the terms page was already imported).
- Added some entries to the theme file.
- Updated footer links.
  
## How to test 🧪
  <Please explain how to test your changes>

- Check out the pages.
- In order to see the careers page with mocked careers, revert this commit locally: 4bf77bb
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [x] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

**Comment by pacxiu** [comment]
Images doesn't seem to load for the team page - I think there is some helper function for linking static files, as we have this `/borrow` added to some paths.
image

File: features/landing/LandingView.tsx [hunk 1]
```diff
@@ -132,7 +132,7 @@ export function Hero({ sx, isConnected }: { sx?: SxStyleProp; isConnected: boole
       }}
     >
       <WelcomeAnnouncement />
-      <Heading as="h1" variant="header2" sx={{ fontSize: 40, mb: 3 }}>
+      <Heading as="h1" variant="header1s" sx={{ mb: 3 }}>
```
Comment (pacxiu) [inline features/landing/LandingView.tsx:135]: I think this variant could be renamed to `header1Marketing` `s` doesn't provide much context about variant

File: pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        >
+          {t('about.heading')}
+        </Heading>
+        <Text variant="light">{t('about.description')}</Text>
+      </Box>
+      <AppLink
+        href="/careers"
+        sx={{ color: 'text.focused', display: 'flex', alignItems: 'center', mt: 3 }}
+      >
+        <Text variant="paragraph1" sx={{ color: 'text.focused', fontWeight: 600 }}>
```
Comment (pacxiu) [inline pages/about.tsx:33]: 600 could be swapped to `semiBold`

File: pages/careers/[slug].tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    <Box sx={{ width: '100%', mt: 4 }}>
+      <AppLink
+        href="/careers"
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          color: 'text.focused',
+        }}
+      >
+        <Icon name="arrow_left" size="16px" sx={{ mr: 1 }} />
```
Comment (pacxiu) [inline pages/careers/[slug].tsx:26]: `size` could be `3` isntead of `16px`

File: pages/careers/[slug].tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        <Text variant="paragraph2" sx={{ color: 'text.focused', fontWeight: 600 }}>
```
Comment (pacxiu) [inline pages/careers/[slug].tsx:27]: 600 could be swapped to `semiBold`

File: pages/careers/[slug].tsx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+            mt: 5,
+          },
+          'p, ul': { variant: 'text.light', my: 2 },
+          ul: { pl: 4 },
+          li: { mb: 3 },
+          strong: {
+            fontWeight: 400,
+            color: 'primary',
+            a: {
+              fontWeight: 600,
```
Comment (pacxiu) [inline pages/careers/[slug].tsx:55]: 600 could be swapped to `semiBold`

File: pages/careers/[slug].tsx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+          h2: {
+            variant: 'text.header3',
+            mb: 3,
+            mt: 5,
+          },
+          'p, ul': { variant: 'text.light', my: 2 },
+          ul: { pl: 4 },
+          li: { mb: 3 },
+          strong: {
+            fontWeight: 400,
```
Comment (pacxiu) [inline pages/careers/[slug].tsx:52]: 600 could be swapped to `normal`

File: pages/careers/index.tsx [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+  )
+}
+
+function RoleCount({ count, sx }: { count: number; sx?: SxStyleProp }) {
+  return (
+    <Flex
+      sx={{
+        width: '26px',
+        height: '26px',
+        borderRadius: '13px',
```
Comment (pacxiu) [inline pages/careers/index.tsx:107]: I would use design roken which is the closest here, I believe we have one for `12px`

File: pages/careers/index.tsx [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+        backgroundColor: 'primary',
+        alignItems: 'center',
+        justifyContent: 'center',
+        flexShrink: 0,
+        ...sx,
+      }}
+    >
+      <Text sx={{ fontSize: 1, fontWeight: 600, color: '#FFF' }}>{count}</Text>
```
Comment (pacxiu) [inline pages/careers/index.tsx:115]: 600 could be swapped to `semiBold` and `#FFF` also with corresponding color token

File: components/Layouts.tsx [hunk 9]
```diff
@@ -16,6 +18,7 @@ interface BasicLayoutProps extends WithChildren {
 
 export interface MarketingLayoutProps extends WithChildren {
   variant?: string
+  topBackground?: 'default' | 'light' | 'lighter' | 'none'
```
Comment (pacxiu) [inline components/Layouts.tsx:21]: I think you could type it `keyof typeof marketingBackgrounds` so you don't have to update those 2 together

**Review by pacxiu — CHANGES_REQUESTED** [review]

File: features/content/privacy/es.mdx [hunk 10]
```diff
@@ -0,0 +1,100 @@
+# Política de Privacidad
+
+*Última actualización: 2 de noviembre de 2020*
```
[the remark this answers is not in the document]
Comment (ferni) [reply features/content/privacy/es.mdx:3]: I think the translations would fall out of scope of this PR, its purpose is just to move the pages, but yeah I can ask if you want.

File: features/landing/LandingView.tsx (hunk 1 above)
Comment (ferni) [reply features/landing/LandingView.tsx]: I did a search and "header1" was not being used. I replaced it so "header1s" is now just "header1".

File: components/Layouts.tsx (hunk 9 above)
Comment (ferni) [reply components/Layouts.tsx:21]: Oh neat!

**Review by pacxiu — APPROVED** [review]

**Pull request merged**